### ISSUE TITLE
Can't select text in messages buffer

### ISSUE THREAD

**Issue by clason** [qvacua/vimr#563]
As opposed to terminal `nvim`, text in the `:messages` list is not selectable (and hence cannot be copied). Needless to say, this is irksome when trying to debug something...

**Comment by ELLIOTTCABLE** [comment]
@clason to be fair, it's *also* not enabled in terminal-vim, if the mouse is gobbled by Vim.

That said, I'm :+1: here — this behaviour can certainly be improved upon in a GUI. The “select and copy to system-clipboard” functionality needn't involve any Vim events or evaluation (that obviously being the reason it's disabled in Vim itself), so I personally think this additional functionality belongs in VimR.

(Of immediate use — I asked a question about this very subject [on SuperUser](https://superuser.com/questions/1257942/how-can-i-get-a-navigable-editable-buffer-with-vims-messages); a kind someone directed me to their invaluable [`:Bufferize` plugin](https://github.com/AndrewRadev/bufferize.vim). A stopgap, to be sure, but …)

**Comment by clason** [comment]
@ELLIOTTCABLE Good point, I didn't realize that I'm relying purely on the terminal's copy&paste feature there.

**Comment by ELLIOTTCABLE** [comment]
Addtl. note — MacVim does exactly this; so there's user-expectation across all the available platforms. `(=`

**Comment by mrinterweb** [comment]
I'm 👍 for this feature as well. It would be very useful to be able to select text from the message buffer. I frequently do this in MacVim, and am really missing the ability to select/copy text.
